static/js/notifications.js

```diff
@@ -277,6 +277,38 @@ exports.notify_above_composebox = function (note, link_class, link_msg_id, link_
     $('#out-of-view-notification').show();
 };
 
+// Set the inline_notification_flow_supported flag to notify
+// desktop app that the current version of zulip app will send inline
+// notification data; the code that sends the data is in process_notification
+// function below. And also listen for send_inline_notification_message.
+if (window.electron_bridge !== undefined) {
+    window.electron_bridge.inline_notification_flow_supported = true;
+
+    // This is done so webapp doesn't have to depend on channel.post
```

Comment:
You mean so the desktop app doesn't have to depend on that, right?

Reply:
Oops, Yes! Fixed.